src/components/Dashboard.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+const Dashboard = (props) => {
+    const [reviews, setReviews] = useState([]);
+    useEffect(() => {
+        fetch('https://pacific-citadel-03998.herokuapp.com/api/reviews')
+        .then(res => res.json())
+        .then((data) => {
+            console.log(data);
+            setReviews(data.reviews);
+        })
+    }, []);
+    return (
+    <ul>
+        {reviews.map((item) => (
+            <div class="review-box">
```

Comment:
as we're in a `<ul>` tag, we should be rendering an `<li>` tag within the map and giving that a unique key which will get rid of the error you are seeing regarding 'each list item should have a unique key'